client/src/components/SignIn/SignIn.scss

```diff
@@ -23,7 +22,7 @@
 
 // Left part;
 .sign-in .sign-in__info {
-    display: none;
+    display: none !important;
     width: 100%;
```

Comment:
!important, remove it by making more specific selector, or add comment with explanations

Reply:
I tryed but they couldn't help me...